tce/src/bintools/BlocksTranslator/include/tinyxml2.h

```diff
@@ -0,0 +1,2495 @@
+/*

```

Comment:
Why is this header included here? Is the license compatible with MIT?

Reply:
fixed!